apps/api/src/modules/notifications/schedule/scheduler.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+SCRIPT_PATH="$( cd "$(dirname "${BASH_SOURCE[0]}")" || exit ; pwd -P )"
+ENV=$SCRIPT_PATH/../../../../.env
+SCHEDULE_TIME=5
+
+source "${SCRIPT_PATH}/../../../../.env"
+
+BASE_URL="http://localhost:${SERVER_PORT}/notifications"
+
+add_notifications_to_queue() {
+  curl -X POST "${BASE_URL}/queue" -H "Content-Type: application/json" -d '{}'
```

Comment:
Did you run this in Docker?
I would say add a step in Docker to install the `curl`.

Reply:
As mentioned in description, I did test this in docker and the shell script executed without any errors